javascript/javascript1/week1/script.js

```diff
@@ -1,8 +1,28 @@
 // Age-ify (A future age calculator)
-let yearOfBirth = 1985;
-let yearFuture = 2025;
+    let yearOfBirth = 1985;
+    let yearFuture = 2025;
+
+    // Used const as formula is fixed.
+    const age = yearFuture - yearOfBirth;
+
+    console.log(`You will be ${age} years old in ${yearFuture}`);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+    let dogYearOfBirth = 2010;
+    let dogYearFuture = 2022;
+    let dogYear = dogYearFuture - dogYearOfBirth;
+
+    // shouldShowResultInDogYears - true - will show dog age in dog years.
+    // shouldShowResultInDogYears - false - will show dog age in human years.
+ 
+    let shouldShowResultInDogYears = true;
+    let dogAgeInDogYear = dogYear * 7;
+
```

Comment:
It is very good that you used template literals. Also good idea with leaving comments. The only thing is in must of the variables you could have used `const` instead of `let`, because you will not reassign them later.

Reply:
let shouldShowResultInDogYears = true;
I used let here, because the value can also be false.
When its changed to false, the dog age in human years is visible in console.

let dogAgeInDogYear = dogYear * 7;
I will change it to const.